python/lsst/validate/drp/matchreduce.py

```diff
@@ -205,6 +206,12 @@ def _loadAndMatchCatalogs(self, repo, dataIds, matchRadius,
 
         ccdKeyName = getCcdKeyName(dataIds[0])
 
+        # Hack to support raft and sensor 0,1 IDs as ints for multimatch
+        if ccdKeyName == 'sensor':
+            ccdKeyName = 'raft_sensor_int'
+            for vId in dataIds:
```

Comment:
I don't think this can work.  Aren't you shadowing the dictionary you are trying to assign with the iterator variable?
Nevermind, I see now.  you are updating the data ids.  Disregard the above.